### PR TITLE
upd(qtile-git): fix installation error

### PR DESCRIPTION
**Pull request by Ruturajn**
- Fixed installation error (unable to verify version from `pyproject.toml`) for `qtile-git`.
- Updated dependencies.

### CODE REVIEW

File: packages/qtile-git/qtile-git.pacscript [hunk 1]
```diff
@@ -21,8 +21,9 @@ build() {
 
 install() {
   sudo pip3 install --upgrade --no-cache --force-reinstall cairocffi cffi xcffib
-  sudo pip3 install dbus-next
-  sudo mkdir -p /usr/src/pacstall/qtile-git/usr/share/xsessions
-  sudo python3 setup.py install --root=/usr/src/pacstall/qtile-git/
-  sudo cp ./resources/qtile.desktop /usr/src/pacstall/qtile-git/usr/share/xsessions/qtile.desktop
+  sudo pip3 install dbus-next setuptools-scm setuptools
```
Comment (Henryws) [inline packages/qtile-git/qtile-git.pacscript:24]: Uninstall this with `removescript`

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/qtile-git/qtile-git.pacscript (hunk 1 above)
Comment (Ruturajn) [reply packages/qtile-git/qtile-git.pacscript:24]: Ok

**Review by Henryws — APPROVED** [review]

**Comment by Ruturajn** [comment]
I wanted to ask, for the `qtile-extras` package, `qtile` is a dependency, so I should just say `pacdeps=("qtile-git")` in the `qtile-extras` pacscript, right?

**Comment by Henryws** [comment]
Yes.

**Comment by Ruturajn** [comment]
Ok, thanks

**Pull request merged**